src/OSPSuite.Presentation/Presenters/Charts/CurveMultiItemEditorPresenter.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OSPSuite.Core.Chart;
+using OSPSuite.Core.Domain;
+using OSPSuite.Presentation.Views.Charts;
+using OSPSuite.Utility;
+
+namespace OSPSuite.Presentation.Presenters.Charts
+{
+   public interface ICurveMultiItemEditorPresenter : IDisposablePresenter, IPresenter<ICurveMultiItemEditorView>
+   {
+      SelectedCurveValues GetSelectedValues();
+      IEnumerable<bool?> AllBooleanOptions { get; }
+      IEnumerable<LineStyles?> AllLineStyles { get; }
+      IEnumerable<Symbols?> AllSymbols { get; }
+      void SetColorChangedFlag();
+   }
+
+   public class SelectedCurveValues
+   {
+      public Color Color { get; set; }
+      public LineStyles? Style { get; set; }
+      public Symbols? Symbol { get; set; }
+      public bool? Visible { get; set; }
+      public bool? VisibleInLegend { get; set; }
+      public bool IsColorSet { get; set; } = false;
```

Comment:
@georgeDaskalakis I do not understand this flag

Reply:
@msevestre thanks for all the other points and corrections! Again this flag has to do with not being able to bind the Color? property to the ColorEdit Control.

Reply:
I see. Probably we could add support for Color? in DevExpressBinder. Merging for now